### DISCUSSION TITLE
CSB Images converted to Absolute Paths ?

### DISCUSSION THREAD

**Discussion by ewuweblab** [codesandbox/codesandbox-client#4966] [General] [answered]
Why do assets originally imported into CodeSandbox, then connected to GitHub, change to an absolute path? Am I using Assets on CodeSandbox incorrectly? Thanks.

Repo : https://github.com/ewuweblab/html-css-only
File: tree.jpg

`https://uploads.codesandbox.io/uploads/user/febe4d5c-3e95-4a86-9ed5-dcfe6abf2eeb/I9Z5-tree.jpg`

**Comment by siddharthkp** [comment]
Hi!

This is the way we handle assets in the sandbox to make them accessible in the sandbox. 

However, @christianalfoni is working on a fix to commit the actual assets to the repo 😄

**Reply by ewuweblab** [reply]
Hi, can you elaborate on: "...make them accessible in the sandbox." Permissions? `public` folder? It's the `static` template with no build process. 

Thanks!

**Comment by siddharthkp** [comment, marked as answer]
The fix is deployed 🚀 

It should commit your assets as their original format now. Feel free to comment back on this discussion if there are issues you run into.

**Comment by ewuweblab** [comment]
It worked on initial commit, then stopped. Sorry. Please see: 

* [https://github.com/ewuweblab/another-csb-test](https://github.com/ewuweblab/another-csb-test)
* Files: `sun.gif` and `test.png`

`https://uploads.codesandbox.io/uploads/user/febe4d5c-3e95-4a86-9ed5-dcfe6abf2eeb/qTYc-sun.gif`

**Reply by siddharthkp** [reply]
Getting a 404 on the repo you mentioned. Can you share another link?

**Reply by ewuweblab** [reply]
* https://github.com/ewuweblab/tree
* `tree.jpg`

Thank you for your help. Please mark this thread as answered. There are simply too many errors and inconsistencies to list. I hope to return to CodeSandbox at a later date. Good luck with your work.

**Reply by siddharthkp** [reply]
Hi!

Thanks for your patience! I know this is frustrating and has taken us a while to catch all the edge cases. 

We deployed added another fix yesterday. So, now when you commit a new image, you should see it in the correct format.

I created this pull request using codesandbox: https://github.com/ewuweblab/tree/pull/1

Please don't hesitate to reach back if you run into it again!
